src/components/ChatApp/MessageListItem/helperFunctions.react.js

```diff
@@ -134,8 +134,9 @@ export function imageParse(stringWithLinks){
     let checkmatch = item.match(replacePattern);
     if(checkmatch){
       result.push(
-        <img key={key} src={checkmatch}
-          style={{width:'95%',height:'auto'}} alt=''/>);
+        <div key={key} style={{width:'300px',height:'250px'}}>
```

Comment:
Changes do work, what if an image is of greater height, it will be compressed, which won't be good in cases when an image is of say 1000px in height.

Reply:
@anshumanv  This is the only solution I could figure out and it worked for most of the searches I made. Please suggest any other way.